src/international-phone-number.coffee

```diff
@@ -73,7 +73,6 @@ angular.module("internationalPhoneNumber", []).directive 'internationalPhoneNumb
       if value
         validity = element.intlTelInput("isValidNumber")
         ctrl.$setValidity 'international-phone-number', validity
-        ctrl.$setValidity '', validity
       else
         value = ''
```

Comment:
It is indeed better to use the new $validators pipeline, but unfortunately that would bump your minimum Angular version from 1.2 to 1.3. Would you consider merging this and then if the $validators approach is desired, add it as a breaking change in a 1.0.0 or 0.1.0 version?